test/test_support.ts

```diff
@@ -305,6 +305,7 @@ export function addDiffMatchers() {
   // tslint:disable:no-require-imports
   jasmine.addMatchers(require('jasmine-diff')(jasmine, {
     colors: true,
+    inline: true,
```

Comment:
I notice that the diff output of this library is less satisfying than the old one, sigh.